TechnicalDocuments/Proposals/BuildersToFunctions.md

```diff
@@ -74,88 +78,74 @@ Given the Module where it's located it could be migrated to:
 
 ```swift
 extension Builder {
-    enum ClinicalRecords {
-        static func makeMedicalHistory(modal: Flow) -> UIViewController
+    struct ClinicalRecords {
+        internal(set) var makeMedicalHistory: (Flow) -> UIViewController = { _ in UIViewController() }
```

Comment:
Unfortunate negative effect of that, as I was mentioning as cons, is that now on call site we wouldn't be able to use named arguments. Static functions don't have it. We could solve it, if needed, with adding a method wrapping such closure:

```swift
struct ClinicalRecords {
    internal(set) var makeMedicalHistory: (Flow) -> UIViewController = { _ in 
        return UIViewController() 
    }
    func makeMedicalHistory(flow: Flow) -> UIViewController {
        return self.makeMedicalHistory(flow)
    }
}
```
I haven't compiled this code but I don't think it will result in ambiguity or something. This is a bit of boilerplate so maybe not worth it.